themes/mongodb/static/navbar.js

```diff
@@ -204,6 +204,69 @@ $(function() {
             })();
         }
     }
+    
+
+    // Create tab functionality for code examples
+    function setupTabs() {
+        
+        // Check if the user has a preference stored, if so load it
+        if (localStorage.getItem("languagePref")) {
+            
+            var currentAttrValue = localStorage.getItem("languagePref");
+            
+            // Show the appropriate tab content and mark the tab as active
+            showHideTabContent(currentAttrValue);
+            showHideSelectedTab(currentAttrValue);
+        }
+        
+        document.querySelectorAll('.tabs .nav-tabs a').forEach(function(element) {
+            element.onclick = function(e) {
+                
+                // Get the href of the clicked tab
+                var currentAttrValue = this.getAttribute('href');
```

Comment:
I would prefer using `element` rather than `this`. The dynamic scoping of `this` is a very unfortunate feature of JavaScript, and is generally best avoided when possible